File: 2271-rearrange-array-elements-by-sign/2271-rearrange-array-elements-by-sign.java
class Solution {
    public int[] rearrangeArray(int[] nums) {
        int[] res = new int[nums.length];
        int a=0;
        int b=1;
        for(int i:nums){
            if(i>0){
                res[a]=i;
                a+=2;
            }else{
                res[b]=i;
                b+=2;
            }
        }
        return res;
    }
}

File: 2271-rearrange-array-elements-by-sign/2271-rearrange-array-elements-by-sign.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        res = [0] * len(nums)
        a = 0
        b = 1
        for i in nums:
            if i > 0:
                res[a] = i
                a += 2
            else:
                res[b] = i
                b += 2
        return res


        